samples/GraphQL.DataLoader.Sample.Default/DealershipDbContext.cs

```diff
@@ -0,0 +1,90 @@
+using GraphQL.DataLoader.Di.Sample.Types;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+
+namespace GraphQL.DataLoader.Di.Sample;
+
+public class DealershipDbContext : DbContext
+{
+    public DbSet<Car> Cars => Set<Car>();
+    public DbSet<Salesperson> Salespeople => Set<Salesperson>();
+
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+    {
+        optionsBuilder
+            .UseSqlite(
+                new SqliteConnectionStringBuilder()
+                {
+                    Mode = SqliteOpenMode.ReadWriteCreate,
+                    BrowsableConnectionString = true,
+                    DataSource = "my.db"
+                }
+                .ToString()
+            )
+            .UseSeeding((ctx, _) =>
+            {
+                if (ctx.Set<Salesperson>().Any())
+                    return;
+                ctx.Set<Salesperson>().AddRange(_seedSalespeople);
+                ctx.SaveChanges();
+            })
+            .UseAsyncSeeding(async (ctx, _, cancellationToken) =>
+            {
+                if (await ctx
+                        .Set<Salesperson>()
+                        .AnyAsync(cancellationToken)
+                        .ConfigureAwait(false))
+                    return;
+                await ctx
+                    .Set<Salesperson>()
+                    .AddRangeAsync(_seedSalespeople, cancellationToken)
+                    .ConfigureAwait(false);
```

Comment:
```suggestion
                ctx.Set<Salesperson>()
                    .AddRange(_seedSalespeople);
```
Calls to Add generally should not be async.  See MS docs.

> This method is async only to allow special value generators, such as the one used by 'Microsoft.EntityFrameworkCore.Metadata.SqlServerValueGenerationStrategy.SequenceHiLo', to access the database asynchronously. For all other cases the non async method should be used.